csharp/ql/test/library-tests/dataflow/library/FlowSummaries.expected

```diff
@@ -1,3 +1,66 @@
+| Microsoft.Extensions.Primitives;StringTokenizer;false;GetEnumerator;();;Element of Argument[-1];Property[System.Collections.Generic.IEnumerator<>.Current] of ReturnValue;value |
```

Comment:
Why were these not in the output before?

Reply:
The struct StringTokenizer stub has just been included as a part of Microsoft.Extensions.Primitives (note that these are derived summaries).

Reply:
Why has `FlowSummariesFiltered.expected` then changed?

Reply:
Microsoft.Extensions.Primitives was included because it contains the StringValues struct. Before the FlowSummaries test didn't know about this struct and couldn't generate any test output for it.